### PR TITLE
feat(icons): Adds LE trophy and certificate filled icons

### PR DESCRIPTION
**Pull request by timjenkins**
### Overview
Adds `TropyFilledIcon` to be used in the progress indicator of the new Path Syllabus page.

Screen Shot 2021-06-03 at 11 48 41 AM

Adds `CertificateFilledIcon` to be used on the Path Syllabus page to reflect the achievement that can be earned on completion.

Screen Shot 2021-06-03 at 11 50 20 AM

Adds `TrophyFilledIcon` and `CertificateFilledIcon` for Syllabus upgrade

### PR Checklist

- [x] Related to designs: [Syllabus Upgrade Figma](https://www.figma.com/file/nWkEaTqiV12SGbbZS0Gmr4/Syllabus-Improvements?node-id=573%3A1)
- [x] Related to JIRA ticket: LX-4800
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.1.0-alpha.a47584.0<br/>@codecademy/gamut-kit@0.4.8-alpha.a47584.0<br/>@codecademy/gamut-labs@19.0.1-alpha.a47584.0<br/>@codecademy/gamut@34.0.1-alpha.a47584.0<br/>@codecademy/styleguide@41.1.0-alpha.a47584.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b91d5231f0321a75edbab5--gamut-preview.netlify.app>https://60b91d5231f0321a75edbab5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b91d5231f0321a75edbab5>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path
+    d="M12 0C9.62663 0 7.30655 0.703788 5.33316 2.02236C3.35977 3.34094 1.8217 5.21508 0.913451 7.4078C0.00519966 9.60051 -0.232441 12.0133 0.230582 14.3411C0.693604 16.6689 1.83649 18.807 3.51472 20.4853C5.19295 22.1635 7.33115 23.3064 9.65892 23.7694C11.9867 24.2324 14.3995 23.9948 16.5922 23.0865C18.7849 22.1783 20.6591 20.6402 21.9776 18.6668C23.2962 16.6934 24 14.3734 24 12C23.9966 8.81843 22.7313 5.76814 20.4816 3.51843C18.2319 1.26873 15.1816 0.00336338 12 0V0ZM12 4.94118C13.3961 4.94118 14.7609 5.35517 15.9217 6.1308C17.0825 6.90643 17.9872 8.00887 18.5215 9.2987C19.0558 10.5885 19.1956 12.0078 18.9232 13.3771C18.6508 14.7464 17.9785 16.0041 16.9913 16.9913C16.0041 17.9785 14.7464 18.6508 13.3771 18.9232C12.0078 19.1956 10.5885 19.0558 9.29871 18.5215C8.00888 17.9872 6.90644 17.0825 6.13081 15.9217C5.35517 14.7609 4.94118 13.3961 4.94118 12C4.94305 10.1285 5.68734 8.3341 7.01073 7.01072C8.33411 5.68734 10.1285 4.94304 12 4.94118V4.94118Z"
+    fill="#10162F" />
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg:4]: i think we want fill to be set to `currentColor` -- correct me if i'm wrong @codecaaron !

File: packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg:2]: pls add a `title` (the other icon is ✨)

**Review by dreamwasp — CHANGES_REQUESTED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.15.0-alpha.d68eb0.0<br/>@codecademy/gamut-kit@0.4.7-alpha.d68eb0.0<br/>@codecademy/gamut-labs@18.4.1-alpha.d68eb0.0<br/>@codecademy/gamut@33.4.4-alpha.d68eb0.0<br/>@codecademy/styleguide@40.4.0-alpha.d68eb0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ba7679c6098400b484976e--gamut-preview.netlify.app>https://60ba7679c6098400b484976e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ba7679c6098400b484976e>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <title>certificate star</title>
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg:2]:
```suggestion
  <title>certificate-star</title>
```
this is just the naming convention we

**Review by dreamwasp — APPROVED** [review]
✨

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.1.0-alpha.e4df5e.0<br/>@codecademy/gamut-kit@0.4.8-alpha.e4df5e.0<br/>@codecademy/gamut-labs@19.0.1-alpha.e4df5e.0<br/>@codecademy/gamut@34.0.1-alpha.e4df5e.0<br/>@codecademy/styleguide@41.2.0-alpha.e4df5e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ba7ea325a1b00d4d7c8974--gamut-preview.netlify.app>https://60ba7ea325a1b00d4d7c8974--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ba7ea325a1b00d4d7c8974>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.1.0-alpha.372ebb.0<br/>@codecademy/gamut-kit@0.4.8-alpha.372ebb.0<br/>@codecademy/gamut-labs@19.0.1-alpha.372ebb.0<br/>@codecademy/gamut@34.0.1-alpha.372ebb.0<br/>@codecademy/styleguide@41.2.0-alpha.372ebb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60babf0f74244f65f9f51646--gamut-preview.netlify.app>https://60babf0f74244f65f9f51646--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60babf0f74244f65f9f51646>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <title>certificate-star</title>
```
Comment (codecaaron) [inline packages/gamut-icons/src/svg/regular/certificate-filled-icon.svg:2]: These titles actually get stripped out by SVGR and renamed, so I like to keep these as the slug from [streamline](https://app.streamlinehq.com/icons/streamline-bold) `discount-star-premium` as ~some~ source of truth level.

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60be2765c58849a2c0dff778--gamut-preview.netlify.app>https://60be2765c58849a2c0dff778--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60be2765c58849a2c0dff778>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.1.0-alpha.2e36e5.0<br/>@codecademy/gamut-kit@0.4.9-alpha.2e36e5.0<br/>@codecademy/gamut-labs@19.0.2-alpha.2e36e5.0<br/>@codecademy/gamut@34.0.2-alpha.2e36e5.0<br/>@codecademy/styleguide@41.2.0-alpha.2e36e5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60be2d7d515884cf62102707--gamut-preview.netlify.app>https://60be2d7d515884cf62102707--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60be2d7d515884cf62102707>Deploy Logs</a></p>

**Pull request merged**